modules/pricer/src/main/java/com/opengamma/strata/pricer/common/GenericVolatilitySurfacePeriodParameterMetadata.java

```diff
@@ -0,0 +1,433 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.pricer.common;
+
+import java.io.Serializable;
+import java.time.Period;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+import org.joda.beans.Bean;
+import org.joda.beans.BeanBuilder;
+import org.joda.beans.BeanDefinition;
+import org.joda.beans.ImmutableBean;
+import org.joda.beans.ImmutablePreBuild;
+import org.joda.beans.JodaBeanUtils;
+import org.joda.beans.MetaProperty;
+import org.joda.beans.Property;
+import org.joda.beans.PropertyDefinition;
+import org.joda.beans.impl.direct.DirectFieldsBeanBuilder;
+import org.joda.beans.impl.direct.DirectMetaBean;
+import org.joda.beans.impl.direct.DirectMetaProperty;
+import org.joda.beans.impl.direct.DirectMetaPropertyMap;
+
+import com.opengamma.strata.collect.tuple.Pair;
+import com.opengamma.strata.market.option.Strike;
+import com.opengamma.strata.market.param.ParameterMetadata;
+
+/**
+ * Surface node metadata for a generic volatility surface node with a specific period to maturity and strike.
+ */
+@BeanDefinition(builderScope = "private")
+public final class GenericVolatilitySurfacePeriodParameterMetadata
```

Comment:
This name comes from the coherence to the existing metadata `GenericVolatilitySurfaceYearFractionParameterMetadata`.